src/angularComponent/ngcOmniboxController.js

```diff
@@ -1,5 +1,7 @@
 import {KEY, isVerticalMovement} from '../coreComponent/keyboard.js';
 
+export const INSTANCE_ID = `ngc-omnibox-${new Date().getTime() * Math.random()}`;
```

Comment:
Yea I could, but I feel like we'd then just have to refactor that later when we make our non-angular version. This works just as well and isn't tied to angular. Thoughts?